server/src/main/java/umm3601/Server.java

```diff
@@ -10,6 +10,12 @@
 public class Server {
     public static void main(String[] args) throws IOException {
 
+        // This users looks in the folder `public` for the static web artifacts,
+        // which includes all the HTML, CSS, and JS files generated by the Angular
+        // build. This `public` directory _must_ be somwhere in the classpath;
+        // a problem which is resolved in `server/build.gradle`.
+        staticFiles.location("/public");
```

Comment:
I tried setting the static thing later in the file when I first tried it, as I had no idea where it "belonged". When I put it even a little further down I got an `IllegalStateException` saying that it had to go before any of the pattern matching things like the `get()` statements. I then wondered (but never searched or experimented to discover) what would happen if there was a conflict between a defined route and a static thing; I'm _assuming_ the defined route would win and we only got to static things as a fall back, but I don't have anything to back that up.

I don't get any interesting/surprising log methods from the Spark server in either dev or deploy "modes". It lets me know when a route is matched, but doesn't say anything when, e.g., a static file is served.